react/w-books/src/app/screens/SignUp/styles.module.scss

```diff
@@ -38,13 +38,19 @@
 }
 
 .signup-green-button,
+.signup-green-button-loading,
 .signup-grey-button {
   background: $green;
   border-radius: 15px;
   color: $white;
+  margin-top: 20px;
```

Comment:
if you want to separate elements that are on top of each other, use margin-bottom, or if you detach an element from the borders, apply a padding on the container parent